lib/bundle/installer.rb

```diff
@@ -26,16 +27,21 @@ def install(entries)
           arg << entry.options
           Bundle::TapInstaller
         end
-        case cls.install(*arg)
-        when :success
-          puts Formatter.success("#{verb} #{entry.name}")
-          success += 1
-        when :skipped
-          puts "Using #{entry.name}"
-          success += 1
-        else
-          puts Formatter.error("#{verb} #{entry.name} has failed!")
-          failure += 1
+        begin
+          case cls.install(*arg)
+          when :success
+            puts Formatter.success("#{verb} #{entry.name}")
+            success += 1
+          when :skipped
+            puts "Using #{entry.name}"
+            success += 1
+          else
+            puts Formatter.error("#{verb} #{entry.name} has failed!")
+            failure += 1
+          end
+        rescue => e
+          puts Formatter.error("#{verb} #{entry.name} errored: #{e}")
```

Comment:
`raised an exception` rather than `error`ed? I'm not sure the failed/errored distinction is obvious.